### DISCUSSION TITLE
Tried out Whale -- overall awesome, but had some issues w/normalization being to abrupt

### DISCUSSION THREAD

**Discussion by swarthyplacebo** [catright/Wale#32] [General]
I ended up uninstalling, totally up to reinstalling, but now my audio is super, super quiet after uninstall. I've tried restarting. Master volume is maxed. Not sure how to fix this.

**Comment by swarthyplacebo** [comment]
Disregard -- something set my Firefox and Foobar2000 to minimum volume
![image](https://github.com/catright/Wale/assets/7636439/fc8f854f-41b9-4db1-8eee-27b63529e821)

**Reply by swarthyplacebo** [reply]
putting them back to 100 fixed the issue

**Reply by catright** [reply]
WALE is being used if you want to amplifying your audio. so if the default volumes are remained in 100 then it'll be harmful, cause they could sound veeeeery loud.
so WALE reduces the default volumes to minimum, and gain up to appropriate level when there's an audio output.

if you want to uninstall WALE, you need to turn them up manually for now.
it's good to hear you found the way by yourself.

**Reply by swarthyplacebo** [reply]
I was under the impression it was an application that was on top of sounds, not changing them in real time :)

**Comment by swarthyplacebo** [comment]
I felt audio cuttoff/normalization was very aggressive with default settings and I'm not great with figuring this out... is there a youtube tutorial or guide I can use. I saw the wiki but I'm too much of a newb to understand how to adjust settings.

**Reply by swarthyplacebo** [reply]
The primary solution I was looking for is with some movies and television, the audio for voices is way too low, whereas the audio for the music/SFX is blowing out the speakers (not literally, but it's loud). So I'm constantly moving back and forth with the volume slider.

If you believe you could help me out with Whale, I'd love to give it a go. I just need guidance.

**Reply by catright** [reply]
you need to do some try and error if you want to change the things just fit to you.
but you don't know where to start, followings are the basics.

the first thing is the **Target Level**, it's the desired volume level you want to set every apps' output. and if you lower it then you should raise you speaker's volume knob.
you can amplify much lower volumes to proper level if you reduce the **Target Level**. because WALE will reduces every apps' loud audio to much lower level, then you will turn your speaker knob to much louder.

second thing is **Comp Rate**. it's a compression rate. if it's 1 then volumes are linearly reduced or raised. but if it's larger than 1, the louder volumes are reduced more, and vice versa.

**Reply by catright** [reply]
![wale_volumeFunctionTest](https://github.com/catright/Wale/assets/31341582/4bc087de-2ca2-4a1a-9523-8d327981bb25)
This is the graph about **Target Level**, _Limit Level_, and **Comp Rate**. where _r_ is the various **Comp Rate**s.

**Reply by swarthyplacebo** [reply]
This is helpful. I'm ... still confused because I'm not an audio technician :D ... I'm going to give it a go again. Is this view available in Whale? I couldn't find it if it was...
